inference-engine/tests/functional/inference_engine/cnn_network/cnn_ngraph_impl_tests.cpp

```diff
@@ -935,72 +935,286 @@ TEST(CNNNGraphImplTests, addOutputForParameter) {
     }
 }
 
-TEST(CNNNGraphImplTests, AddOutputToExperimentalOp) {
+TEST(CNNNGraphImplTests, AddOutputToExperimentalOpOpset6) {
     std::string model = R"V0G0N(
 <net name="Activation" version="10">
     <layers>
-        <layer name="in1" type="Parameter" id="0" version="opset1">
-            <data shape="1,3,22,22" element_type="f32"/>
-            <output>
+		<layer id="0" name="in0" type="Parameter" version="opset1">
```

Comment:
Reformat the whole file and use 4 spaces instead of one tab character. Then verify that the indentation is correct.

Reply:
Fixed.